clients/payment/src/components/CardPayment.tsx

```diff
@@ -253,7 +252,10 @@ export const CardPayment = ({ paymentLink, paymentMethodId, publicKey }: Props)
                     <View
                       style={[
                         styles.cardLogo,
-                        (isPaymentMethodValid === false || cardNumberState !== "valid") && {
```

Comment:
As you are using this condition 3 times, maybe you could create a variable for it:

```ts
const isNegative = isPaymentMethodValid === false ||
  cardNumberState === "cardNotSupported" ||
  cardNumberState === "empty" ||
  cardNumberState === "invalid";

...

isNegative && { borderColor: colors.negative[500] }
```